pkg/openstack-cinder/config/cloudinfo.go

```diff
@@ -81,6 +82,15 @@ func getCloudInfo() (*CloudInfo, error) {
 	ua := gophercloud.UserAgent{}
 	ua.Prepend(fmt.Sprintf("csi-operator/%s", version.Get().GitCommit))
 
+	// our assets mount the CA file at a known path and we don't want to rely on other things
+	// setting the 'cafile' value in our clouds.yaml file to the same path, so we explicitly
+	// override this if a CA file is present
+	if _, err := os.Stat("/etc/kubernetes/static-pod-resources/configmaps/cloud-config"); err == nil {
+		// NOTE(stephenfin): gophercloud (or rather, the clientconfig package) doesn't currently
+		// provide the way to override configuration other than via environment variables
+		os.Setenv("OS_CACERT", "/etc/kubernetes/static-pod-resources/configmaps/cloud-config")
```

Comment:
Isn't `/etc/kubernetes/static-pod-resources/configmaps/cloud-config` a directory? If so we'll need to point to the file holding the ca-cert...